mailparser/mailparser.py

```diff
@@ -226,8 +226,12 @@ def message_id(self):
 
     @property
     def to_(self):
-        return decode_header_part(
-            self._message.get('to', self._message.get('delivered-to')))
+        to_ = self._message.get('to')        
+        if not to_:
+            return None
+        else:
```

Comment:
Whoops, something got lost while I was managing separate branches, It should be:
```
`     def to_(self):
 -        return decode_header_part(
 -            self._message.get('to', self._message.get('delivered-to')))
 +        to_ = self._message.get('to')        
 +        if not to_:
 +            return None
 +        else:
 +            return decode_header_part(
 +                self._message.get('to', self._message.get('delivered-to')))`
```
In order to handle cases where there is no recipient listed. I found this to be the case in about 1% of some 250k messages.